index.js

```diff
@@ -68,7 +84,11 @@ name.decode = function (buf, offset) {
       if (totalLength > 254) {
         throw new Error('Cannot decode name (name too long)')
       }
-      list.push(buf.toString('utf-8', offset, offset + len))
+      if (mail) {
+        list.push(buf.toString('utf-8', offset, offset + len).replace(/\./g, '\\.'))
+      } else {
+        list.push(buf.toString('utf-8', offset, offset + len))
+      }
```

Comment:
```suggestion
      let label = buf.toString('utf-8', offset, offset + len)
      if (mail) {
        label = label.replace(/\./g, '\\.'))
      }
      list.push(label)
```

Reply:
There was one bracket too many in L92. I commited the changes.

Reply:
Thanks